docs/develop/configure-your-add-in-to-use-a-shared-runtime.md

```diff
@@ -52,6 +50,9 @@ Follow these steps for a new or existing project to configure it to use a shared
     <DefaultSettings>
     ```
 
+> [!NOTE]
+> If your add-in is an Excel add-in with custom functions, be sure to remove the `CustomFunctionsRuntime` requirement.
+
 1. Find the `<VersionOverrides>` section and add the following `<Runtimes>` section. The lifetime needs to be **long** so that your add-in code can run even when the task pane is closed. The `resid` value is **Taskpane.Url**, which references the **taskpane.html** file location specified in the ` <bt:Urls>` section near the bottom of the **manifest.xml** file.
 
     > [!IMPORTANT]
```

Comment:
In the "Runtime lifetime" section, it discusses that "long" is recommended for Excel add-ins. Should Word and PowerPoint also use "long"?

Reply:
It also specifically mentions Excel (as the sample project, etc.). We may want to change that - maybe generically talk about add-in project but not Excel specifically unless required.

Reply:
Yes, I fixed to indicate this applies to Word, Excel, and PowerPoint.